quarkchain/tests/test_config.py

```diff
@@ -181,8 +182,12 @@ def test_serialization(self):
     "BLOCK_REWARD_DECAY_FACTOR": 0.5,
     "ENABLE_TX_TIMESTAMP": null,
     "TX_WHITELIST_SENDERS": [],
-    "ENABLE_EVM_TIMESTAMP": null
-}"""
+    "ENABLE_EVM_TIMESTAMP": null,
+    "TX_POOL_PRICE_THRESHOLD": 1000000000000000000,
```

Comment:
I should suggest using

MIN_TX_POOL_GAS_PRICE

or

MIN_TX_POOL_GAS_PRICE_THRESHOLD

as THRESHOLD can be THRESHOLD for lower or greater values

Reply:
Similar to the previous comment, would suggestion
MIN_MINE_TX_GAS_PRICE

or

MIN_MINE_TX_GAS_PRICE_THRESHOLD